### PR TITLE
Enhancing troubleshoot section with Windows Operational Readiness

### PR DESCRIPTION
**Pull request by knabben**
Adding  Windows operational Readiness instructions for troubleshooting and Windows cluster validation

refs #45130

### CODE REVIEW

File: content/en/docs/concepts/windows/intro.md [hunk 1]
```diff
@@ -415,6 +415,17 @@ plane to manage the cluster it, and nodes to run your workloads.
 
 The Kubernetes [cluster API](https://cluster-api.sigs.k8s.io/) project also provides means to automate deployment of Windows nodes.
 
+### Validating the Windows cluster operability
+
+The Windows Operational Readiness is a sub-project and specification of the SIG Windows,
+which introduces a structured test suite. This suite is split into two sets of tests,
```
Comment (sftim) [inline content/en/docs/concepts/windows/intro.md:421]:
```suggestion
The Kubernetes project provides a _Windows Operational Readiness_ specification,
accompanied by structured test suite. This suite is split into two sets of tests,
```

**Review by sftim — CHANGES_REQUESTED** [review]
Hi. The change being added _isn't_ a troubleshooting guide, and the section being added isn't a deployment tool.

Would you be willing to revise further @knabben?

**Comment by knabben** [comment]
Moving it as a subsection of get help and troubleshooting.

**Comment by geetikabatra** [comment]
LGTM

**Review by sftim — COMMENTED** [review]
LGTM for docs, with a quibble

Can we get a technical review from SIG Windows as well?

**Comment by sftim** [comment]
@geetikabatra - was your LGTM on behalf of SIG Windows?

**Comment by geetikabatra** [comment]
@sftim Yep

**Pull request merged**